src/rvs_util.cpp

```diff
@@ -231,9 +319,17 @@ int display_gpu_info (void) {
     info.node_id   = node_id;
     info.gpu_id    = gpu_id;
     info.device_id = dev_id;
+    auto ret = rsmi_dev_vbios_version_get(i, info.vbios_ver, 1024);// check returns TODO
+    info.fmver = {};
+    fill_firmware_versions(i, info.fmver);
     gpu_info_list.push_back(info);
 
   }
+  std::string rcm{ROCM_PATH};
+  if (rcm.find('/') != std::string::npos){
+	  rcm = rcm.substr(rcm.find_last_of('/')+1);
+  }
+  std::cout  << rcm << " Running on " << getOSName() << std::endl;
```

Comment:
This will display as "rocm Running on ..." ? Not the Rocm version right ?

Reply:
it will show full rocm version as ROCM_PATH has entire path from "/" til end of version. We are taking only the last part of path which is rocm with version.